spec.html

```diff
@@ -26916,58 +26939,64 @@ <h1>Example Cyclic Module Record Graphs</h1>
             <table>
               <thead>
                 <tr>
-                  <th>Module</th>
-                  <th>[[DFSIndex]]</th>
-                  <th>[[DFSAncestorIndex]]</th>
-                  <th>[[Status]]</th>
-                  <th>[[AsyncEvaluation]]</th>
-                  <th>[[AsyncParentModules]]</th>
-                  <th>[[PendingAsyncDependencies]]</th>
+                  <th class="corner-cell">
+                    <span class="column">Fields</span>
+                    <div class="slash"></div>
+                    <span class="row">Modules</span>
+                  </th>
```

Comment:
Nit: shouldn't these headings be singular, as they are elsewhere in the spec at e.g. [ECMAScript Data Types and Values](https://tc39.es/ecma262/multipage/ecmascript-data-types-and-values.html#sec-ecmascript-data-types-and-values) and [Abstract Operations](https://tc39.es/ecma262/multipage/abstract-operations.html#sec-abstract-operations) and the current version of this table?
```suggestion
                  <th class="corner-cell">
                    <span class="column">Field</span>
                    <div class="slash"></div>
                    <span class="row">Module</span>
                  </th>
```